tests/qxmppatmtrustmemorystorage/tst_qxmppatmtrustmemorystorage.cpp

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (C) 2008-2022 The QXmpp developers
+ *
+ * Authors:
+ *  Melvin Keskin
+ *
+ * Source:
+ *  https://github.com/qxmpp-project/qxmpp
+ *
+ * This file is a part of QXmpp library.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ */
+
+#include "QXmppAtmTrustMemoryStorage.h"
+#include "QXmppConstants.cpp"
+#include "QXmppConstants_p.h"
+#include "QXmppTrustMessageKeyOwner.h"
+
+#include "util.h"
+
+class tst_QXmppAtmTrustMemoryStorage : public QObject
```

Comment:
It would be nice to have this in the same test class as trustmemorystorage, because the unit tests really slow down building.